pkg/playback/http_replayer.go

```diff
@@ -32,14 +32,14 @@ func newReplayServer(webhookURL string) (httpReplayer replayServer) {
 	return httpReplayer
 }
 
-// Read a previously recorded cassette into the replayer and ready it for replaying.
-func (httpReplayer *replayServer) readCassette(reader io.Reader) error {
+// Reads a cassette file, decodes it with serializer and loads it in the cassette.
+func (httpReplayer *HTTPReplayer) readCassette(reader io.Reader) error {
 	// TODO(DX-5701): We may want to allow different types of replay matching in the future (instead of simply sequential playback)
 	sequentialComparator := func(req1 interface{}, req2 interface{}) (accept bool, shortCircuitNow bool) {
 		return true, true
 	}
```

Comment:
@richardm-stripe I'm a little confused at this here - it always just returns [true, true] is that expected? Is that a pending task that has to be done?

Reply:
ah so does that imply that there is a cassette file _per_ test? The concern here would be that running a single test (instead of the whole suite), the replayer would just return the first Interaction, no matter what request/test it corresponds to.

Reply:
for the matching on non-deterministic calls, yeah I get the issue - it's interesting, I just looked at ruby VCR how they handle that and there is an option to set on what to match a request on, but by default it's url and method (https://github.com/vcr/vcr/blob/master/lib/vcr/request_matcher_registry.rb)

Reply:
Yes there's (at least) two distinct ways you can use VCR -- one is "one cassette per test" where the cassette kind of represents a sequence of expected actions. The other is as kind of a fancy database of recorded mocks, and the same "response" will be returned for potentially many requests inside a single test suite. My intuition is that making "sequence of expected actions" the happy path is probably the right way to go for stripe playback, at least at first?

Reply:
Also see https://github.com/stripe/stripe-playback-node our "wrapper" for this. We wrap the test runner's `it` in order to intercept the user's test description to generate an appropriate cassette name: https://github.com/stripe/stripe-playback-node/blob/master/index.js#L6-L7